lib/tasks/rubocop.rake

```diff
@@ -1,7 +1,10 @@
-require 'rubocop/rake_task'
+begin
+  require 'rubocop/rake_task'
 
-desc 'Run style checker'
-RuboCop::RakeTask.new(:rubocop) do |task|
-  task.requires << 'rubocop-rspec'
-  task.fail_on_error = true
+  desc 'Run style checker'
+  RuboCop::RakeTask.new(:rubocop) do |task|
+    task.requires << 'rubocop-rspec'
+    task.fail_on_error = true
+  end
+rescue LoadError
 end
```

Comment:
👎  I'd say, not being there at all is a good enough indication (and otherwise you'll get this message when you run any rake task without the gem available) .